.github/workflows/build.yml

```diff
@@ -59,8 +92,10 @@ jobs:
       with:
         name: Test logs ${{ matrix.name }}
         path: |
-          tests/test.sh.log
+          tests/*.log
           tests/openssl.cnf
-          tests/tmp/p11prov-debug.log
-          tests/tmp/debugvars
+          tests/tmp.softokn/p11prov-debug.log
+          tests/tmp.softokn/debugvars
+          tests/tmp.softhsm/p11prov-debug.log
```

Comment:
I think you can use tests/tmp.${{ matrix.name }}/ patterns here to pull the logs